docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
 
-##  <a name="install-enterprise"></a>Prerequisites
+**Prerequisites:**
 
-- Depending on your security mode in Enterprise DC/OS, you may [need to provision a service account](https://docs.mesosphere.com/service-docs/spark/spark-auth/) before installing Spark. Only someone with `superuser` permission can create the service account.
-	- `strict` [security mode](https://docs.mesosphere.com/1.9/installing/custom/configuration-parameters/#security) requires a service account.  
-	- `permissive` security mode a service account is optional.
-	- `disabled` security mode does not require a service account.
+- [DC/OS and DC/OS CLI installed](https://docs.mesosphere.com/1.9/installing/).
```

Comment:
yeah, we could simply link to `https://docs.mesosphere.com/installing/`. however, since these service versions only support specific versions of DC/OS, latest won't always be correct as we iterate (e.g. 2 months from now when 1.10 is GA, this version of the spark guide should still link specifically to the 1.9 doc, correct?).